### ISSUE TITLE
Requesting deploymentBase of an finished action marks deployment as unfinished.

### ISSUE THREAD

**Issue by SijmenHuizenga** [eclipse/hawkbit#1085]
If a target requests a deploymentBase action _after_ the target marked the deploymentBase action as 'Finished', the deployment status in hawkbit is set to 'Retrieved'. Trying to mark the deploymentBase action as 'Finished' after this is impossible since the feedback endpoint returns 'Gone'.

In the UI, this looks like this:

![image](https://user-images.githubusercontent.com/8008227/108095772-81c18400-7080-11eb-9dc7-c7bfc4d95b26.png)

I'm guessing that the solution would be to return 'Gone' if a deploymentBase action is being requested that has already been marked as 'finished'.

**Comment by bogdan-bondar** [comment]
Yes indeed, action should not be allowed to change after it had been finished, thanks for reporting it!

**Comment by FeDeGbR89** [comment]
Still present after 2,5 years. Any update?
